src/clr-angular/utils/focus/focus.service.ts

```diff
@@ -72,8 +81,7 @@ export class FocusService {
         // If performance ever matters for this, we can refactor using additional private methods.
         const nextObs = isObservable(next) ? next : of(next);
         nextObs.subscribe(item => {
-          this.moveTo(item);
-          return true;
+          return this.moveTo(item);
```

Comment:
Not a blocker, mabye a nit because it wasn't already here:

Does it make sense to also add the boolean return type here for clarity? Similar to how it was added to moveTo?